src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -79,7 +79,7 @@ public T remove(int index) {
     public boolean remove(T t) {
         Node<T> node = head;
         for (int i = 0; i < size; i++) {
-            if (node.element == t || node.element.equals(t)) {
+            if (node.element == null || node.element == t || node.element.equals(t)) {
```

Comment:
does this check work correctly?
what does happen if node.element == null and t != null

Reply:
should be false, redone